CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * Correctly detect failed version checker Pods
 * retry cluster status updates, reducing test flakes
 
+## Added
+* Add validation webhook to reject changes to additional labels field
```

Comment:
nit: The validating webhook was already in place, can we change this note to say "Update validating webhook to..."?

nit: let's reference the field name here `.Spec.AdditionalLabels`?